docs/tutorial/1-serialization.md

```diff
@@ -45,7 +45,7 @@ We'll need to add our new `snippets` app and the `rest_framework` app to `INSTAL
     INSTALLED_APPS = (
         ...
         'rest_framework',
-        'snippets',
+        'snippets.apps.SnippetsConfig',
```

Comment:
It puzzled me too. Reading a bit more about it, I think the all point is to avoid shady declaration in `__init__.py`.
You only need it if you declare explicitly the `apps.py` module.